src/js/components/UnderflipTeam/UnderFlipTeam.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import TeamConfig from '../config/team-config';
+import TeamMember from '../TeamMember/TeamMember';
+
+const UnderflipTeam = () => TeamConfig
+  .map((item) => (
+    <TeamMember member={item} />
+  ));
```

Comment:
```suggestion
const UnderflipTeam = () => TeamConfig.map((item) => <TeamMember member={item} />);
```
Any reason for making this so many lines?